### PR TITLE
DATALAB-386: /v1/recommendations now comes from Bramble

### PR DESCRIPTION
**Pull request by tatiana**
Additional work:
- changed the Amanita-HTTP /recommendations to /http/recommendation
- added `context` to recommended items from Bramble
- added `description` to Videos, based on short synopsis
- added `duration` to Videos, so the app will be able to display synopsis

This version of Snowdrop requires this version of Amanita:
https://github.com/bbc/connected-data-amanita/pull/24

### CODE REVIEW

File: app/services/amanita.py [hunk 1]
```diff
@@ -39,33 +39,35 @@ def pid_to_uri(pid):
...
-def to_snowdrop(amanita_item, version=True):
+def to_snowdrop(amanita_item):
     """
     Convert a media item dict from Amanita into a Snowdrop
     media item dict.
 
     Sample output:
     {
+        "description": "To be or not to be",
+        "id": "p123gas1v",
```
Comment (maria-panteli) [inline app/services/amanita.py]: add duration

File: app/snowdrop.py [hunk 2]
```diff
@@ -34,19 +34,19 @@ def index():
...
 
 
-@app.route('/beta/recommendations', methods=['GET'])
+@app.route('/v1/recommendations', methods=['GET'])
 @authorise
-def beta_home():
+def home():
     """Return recommendations from the Reco System via gRPC."""
     limit = int(request.args.get('limit', settings.DEFAULT_PAGE_LIMIT))
     cookie = request.cookies.get("ckns_atkn")
```
Comment (maria-panteli) [inline app/snowdrop.py]: for some reason getting the cookie like this doesn't work for me:
```
{
"ERROR": "<_Rendezvous of RPC that terminated with (StatusCode.INTERNAL, [BRAMBLE:ValueError] Neither user_cookie or user_id were given)>"
}
```
however it works if I replace this with ```cookie = 'myactualcookie'```

Comment (tatiana) [reply app/snowdrop.py]: Hi @maria-panteli! Thanks for your feedback :)
How did you try it out? Snowdrop requires the user cookie on the header, so in order to test it, you should use something like:
```
curl 'http://localhost:5000/v1/recommendations?limit=1' -H 'Cookie: ckns_atkn=your-cookie'
```
Replacing `your-cookie` by the string generated by `cookie_monster`.

File: tests/snowdrop_test.py [hunk 3]
```diff
@@ -138,16 +143,19 @@ def test_error_config(mock_get, client):
 @patch("app.services.bramble.client.stub.ListRecommendations", return_value=RECOMMENDATIONS)
 @patch("app.services.amanita.session.get", return_value=MockResponse(ITEM, 200))
 def test_beta_recommendations(mock_get, mock_rpc, mock_authorised, client):
```
Comment (felixmercermoss) [inline tests/snowdrop_test.py]: rename to v1 recommendations

**Comment by felixmercermoss** [comment]
Tests pass and all works locally. Changes look sensible and in line with app team suggestions. Looks good :shipit:

File: app/snowdrop.py (hunk 2 above)
Comment (maria-panteli) [reply app/snowdrop.py]: I've been running it wrong! I was checking localhost:5000/v1/recommendations from my browser.  Might be useful to add this curl command in the readme (for newbies like me :)).

**Comment by janae** [comment]
please deploy to stage! :shipit:

**Comment by tatiana** [comment]
@janae The issue in your computer is likely that you have $AMANITA_HOST setup locally - and this is affecting the tests. Could you try to unset it?

**Pull request merged**